src/lib/y2security/selinux.rb

```diff
@@ -122,21 +172,30 @@ def save
       return false unless configurable?
 
       Yast::Bootloader.modify_kernel_params(mode.options)
```

Comment:
Just a note, perhaps the calls to the `Bootloader` calls should be made elsewhere, in the _client_ using it and this method be only responsible to update the needed values in the SELinux configuration file (CFA::Selinux, /etc/selinux/config)

Reply:
I think it is not so important anymore to do this because you have renamed the class to selinux and that could include the bootloader call.